### DISCUSSION TITLE
Video Explicativo

### DISCUSSION THREAD

**Discussion by otachies** [elisei/full-checkout#40] [General]
Boa tarde. Tudo certo?
Vi que o video doyoutube acabou saindo do ar. Tinha visto ele antes de testar a extensão (há uns bons meses atrás) e pareceu bem explicativo. Todas funcionalidades e principalmente os campos de endereço lembro que foram tratados.

Por acaso vocês ainda tem ele disponível?

Abraços e parabéns pelo trabalho.

**Comment by elisei** [comment]
Olá @otachies 

Muito obrigado! 

De fato ele caiu... mas segue!
https://user-images.githubusercontent.com/1786389/161134172-ad604cf4-6875-40dc-8c3a-ddbbc7e46922.mp4

**Comment by otachies** [comment]
Muito obrigado, Bruno!
Posso tirar dúvida por aqui com você?

Pretendo usar o Paypal Plus como meio de pagamento. Você já testou com ele? https://github.com/br-paypaldev/magento2-module
O que vi de requisito é o seguinte:
```
Habilitar o VAT Number no Front-end:
    STORES -> Settings -> Configuration -> Customers -> Customer Configuration -> Create New Account Options -> Show VAT Number on Storefront (Habilitar como "Yes")

Habilitar como obrigatório o Tax/VAT Number no endereço do Cliente:

    STORES -> Settings -> Configuration -> Customers -> Customer Configuration -> Name and Address Options -> Show Tax/VAT Number (Habilitar como "Required")
```
Nesse caso, vou poder usar o Social Login? 

Outra dúvida: a página de criação de conta padrão do magento não é alterada, correto? 
Gostei do layout e a forma que a conta é criada no seu checkout. Ficaria mais bonito do que na página padrão do Magento

**Reply by elisei** [reply]
Olá @otachies!

Não testei ainda, já foram testados:
- [x] Moip
- [x] Getnet
- [x] MercadoPago
- [x] Pagar.me

Todos funcionaram bem.

Sobre o requisito recomendado, ele não faz muito sentido coletar 2 vezes a mesma informação é bem improdutivo... E não o Social Login não irá funcionar corretamente caso tenha a obrigação de coleta do Tax/Vat.

**Comment by otachies** [comment]
Muito obrigado, @elisei 
De fato com o Pagar.me e MercadoPago está funcionando corretamente. Vou fazer alguns testes com o Paypal para que funcione junto com o seu checkout.

Ao menos no meu tema estou tendo 2 probleminhas. Vou procurar se tem algo parecido ou relatar nas issues.
Abraços
